common/openshift.init.yml

```diff
@@ -28,6 +28,12 @@ parameters:
   - name: ORACLEDB_SECRET
     description: Oracle database keystore secret/password
     required: true
+  - name: ORACLEDB_USER_W
```

Comment:
Those two can be removed, as they are redundant. We can make use of just the ORACLEDB_USER and ORACLEDB_PASSWORD.